src/prostredi/editor/clion/cmake.md

```diff
@@ -0,0 +1,92 @@
+## Co je CMake?
+
+CMake je open-source build systém, který automatizuje proces konfigurace a sestavení softwarových projektů napříč různými platformami.
+
+## Co znamená jednotlivý řádek, když vytvořím projekt v CLionu?
+
+Když vytváříme projekt v CLionu, automaticky se vygeneruje soubor **CMakeLists.txt**, který vypadá následovně:
+
+```cmake
+cmake_minimum_required(VERSION 3.21)
+project(testupr C)
+
+set(CMAKE_C_STANDARD 11)
+
+add_executable(testupr main.c)
+```
+
+```cmake
+cmake_minimum_required(VERSION 3.21)
+```
+
+- Tento řádek určuje minimální požadovanou verzi CMake, která je potřebná pro sestavení projektu. V tomto případě je požadovaná verze 3.21. Pokud máte starší verzi CMake, sestavení projektu nebude
+  možné.
+
+```cmake
+project(testupr C)
+```
+
+- Definuje název projektu, který je v tomto případě "testupr", a určuje, že projekt je napsán v jazyce C.
+
+```cmake
+set(CMAKE_C_STANDARD 11)
+```
+
+- Tento řádek nastavuje verzi standardu jazyka C, kterou má kompilátor použít. Zde je to C11.
+
+```cmake
+add_executable(testupr main.c)
+```
+
+- Tento příkaz říká, že výsledkem buildu bude spustitelný soubor nazvaný "testupr".
+
+> Když chcete přidat další soubory do projektu, jednoduše je přidejte takto:
+> ```cmake
+> add_executable(testupr main.c funkce.c funkce.h)
+> ```
+> Pokud soubor vytvoříte přes CLion automaticky vám je zde přidá.
+
+![CMake_přidání_souboru](../../../static/video/cmake_add_file.gif)
+
+## Přidání SDL2 do projektu
```

Comment:
Návod na instalaci SDL už ve skriptech taky máme: https://mrlvsb.github.io/upr-skripta/c/aplikovane_ulohy/sdl.html

Spíše bych to nechal tím směrem, že si stáhnou předkompilované SDL, než do toho tahat git, to je moc komplikované. Ale ten popis SDL + CMake ve skriptech je docela basic, takže vzít ty body 4-6 odsud a přidat je tam by bylo fajn.